tests/regression/gdoc/test_chrome_gdoc_real_input_latency.sikuli/test_chrome_gdoc_real_input_latency.py

```diff
@@ -0,0 +1,39 @@
+# if you are putting your test script folders under {git project folder}/tests/, it will work fine.
+# otherwise, you either add it to system path before you run or hard coded it in here.
+sys.path.append(sys.argv[2])
+import os
+import gdoc
+import shutil
+import browser
+import time
+import json
+
+chrome = browser.Chrome()
+gd = gdoc.gDoc()
+chrome.clickBar()
+chrome.enterLink(sys.argv[3])
+gd.wait_for_loaded()
+
+setAutoWaitTimeout(10)
+sample1_fp = os.path.join(sys.argv[4], sys.argv[5])
+sample2_fp = os.path.join(sys.argv[4], sys.argv[5].replace('sample_1', 'sample_2'))
+os.remove(sample1_fp)
+
+sleep(2)
+capture_width = int(sys.argv[6])
+capture_height = int(sys.argv[7])
+
+t1 = time.time()
+capimg1 = capture(0, 0, capture_width, capture_height)
+type('a')
+sleep(0.1)
```

Comment:
In normal condition, a should appear within 100ms, but if lag happened, that could lead the show up after 100 ms, and that will cause the calculation of AIL much smaller than expected.